package.json

```diff
@@ -40,19 +40,19 @@
     "@angular/compiler-cli": "7.2.15",
     "@angular/language-service": "7.2.15",
     "@types/faker": "^4.1.7",
-    "@types/jasmine": "^2.8.16",
+    "@types/jasmine": "^3.5.0",
     "@types/jasminewd2": "^2.0.8",
     "@types/node": "^6.14.9",
     "codelyzer": "^4.5.0",
-    "coveralls": "^3.0.8",
+    "coveralls": "^3.0.9",
     "faker": "^4.1.0",
-    "jasmine-core": "~2.8.0",
-    "jasmine-spec-reporter": "~4.2.1",
-    "karma": "^2.0.2",
-    "karma-chrome-launcher": "~2.2.0",
-    "karma-coverage-istanbul-reporter": "^1.4.3",
-    "karma-jasmine": "^1.1.2",
-    "karma-jasmine-html-reporter": "^0.2.2",
+    "jasmine-core": "^3.5.0",
+    "jasmine-spec-reporter": "^4.2.1",
+    "karma": "^4.4.1",
+    "karma-chrome-launcher": "^3.1.0",
+    "karma-coverage-istanbul-reporter": "^2.1.1",
+    "karma-jasmine": "^2.0.1",
+    "karma-jasmine-html-reporter": "^1.4.2",
```

Comment:
I had some trouble getting the tests to pass, but I realized it was because the testing tools needed to be upgraded after our Angular upgrade. I think I finally got them all working again.